hpack/table.py

```diff
@@ -0,0 +1,215 @@
+from collections import deque
+
+class HeaderTable(deque):
+    """
+    Implements the combined static and dynamic header table
+
+    The name and value arguments for all the functions
+    should ONLY be byte arrays (b'') however this is not
+    strictly enforced in the interface.
+
+    See RFC7541 Section 2.3
+
+    Class Constants
+        DEFAULT_SIZE : Default maximum size of the dynamic
+                       table. See RFC7540 Section 6.5.2
+        STATIC_TABLE : Constant list of static headers.
+                       See RFC7541 Section 2.3.1 and
+                       Appendix A
+    """
+
+    DEFAULT_SIZE = 4096
```

Comment:
The format for this comment would be:

```python
#: Default maximum size of the dynamic table. See RFC7540 Section 6.5.2.
DEFAULT_SIZE = 4096
```

Except you should pay attention to column width. ;)